### ISSUE TITLE
Plasma clearance parameter

### ISSUE THREAD

**Issue by teutonicod** [Open-Systems-Pharmacology/PK-Sim#1324]
Dear all,

In the unspecified liver clearance process in PK-Sim, the intrinsic clearance for the liver is calculated using several parameters, one of them being referred to as **Plasma clearance**. Do you know how this parameter is defined? In a simple model (IV bolus and only unspecified clearance) the value for this **Plasma clearance** does not coincide with the **Total body clearance** in the PK-Analysis section, do you have an explanation for the difference of these parameters? 
Thanks you in advance for your help!

Best regards,

Donato

**Comment by msevestre** [comment]
@Yuri05 @sfrechen Anything to change here? If not, please close the issue

**Comment by Yuri05** [comment]
>Anything to change here?

From my point of view: not. @sfrechen please comment

**Comment by sfrechen** [comment]
I would say: yes.
Do not correct for f_cell in the CLspec equation.

**Comment by Christoph27** [comment]
I agree, "Total Hepatic Clearance" is currently not doing what I would expect.

**Comment by msevestre** [comment]
As far as I remember, this was added on purpose a while ago to solve another problem. I don't have access to SVN anymore or youtrack but maybe the reason cab be found? 

Was it to ensure that the same value of clspec  in a total plasma clearance process vs intrinsic clearance would produce the same results?

**Comment by Christoph27** [comment]
Might be the reason (I was not involved in that previous discussion) but that is at the cost of an larger deviation of the resulting PBPK clearance from the clearance used as input parameter than needed. Why do we have then the "Total Hepatic Clearance" process at all?

**Comment by teutonicod** [comment]
I agree with @Christoph27 , I think we need to decide if at all keep this "Plasma clearance" entry in the "Total plasma clearance". Since it can be very misleading. If we think that a back calculation of the clearance with the well stirred model can be useful, I would propose to correct the formula and rename the entry. But I am still unsure on the utility of such a back calculation

**Comment by msevestre** [comment]
>it seems that the factor ~1.5 was introduced in order to get the same total plasma clearance with the same specific intrinsic clearance regardless of using a "Total Hepatic Clearance" process or a clearance process with a metabolizing enzyme.

Yes this is correct
If we remove the f_cell, suddenly the same value of CL_Spec would produce two different curves, which is probably more of an issue don't you think?

As to why we have a Total Plasma Clearance process. They are a few reasons
# This was the input that we had in the old PKSim v4.x
# The concept of specific clearance is far from trivial and typically hard to understand for new users. Using a plasma clearance as a first step in learning the program was a requested feature
# If someone has a value for a Total plasma clearance, inputting it in the field can give a rough estimate as to what the CL_spec might be. 

> If we think that a back calculation of the clearance with the well stirred model can be useful, I would propose to correct the formula and rename the entry.

@teutonicod I don't understand what you mean

**Comment by Christoph27** [comment]
Just to clarify: I'm fine with having a "Total Hepatic Clearance" process which uses a well stirred model and a total plasma clearance as input. But what is currently implemented is something else (and probably not what we had in the old PKSim v4.x) with an additional factor ~1.5. I don't see the issue with "same value of CL_Spec would produce two different curves" since the enzyme concentration is implicitly different.

**Comment by msevestre** [comment]
> I don't see the issue with "same value of CL_Spec would produce two different curves"

Not really. If you define an enzyme in the individual with a RelExp of 1 in liver and a Ref Conc of 1 (similar to our undefined liver enzyme created), then the same Cl_spec would produce different results, which is definitely weird. 

Again, I am happy to revert the changes. I just want to make sure that we understand that the changes were added for a specific reason.

**Comment by Yuri05** [comment]
As far as I understood @sfrechen : [his suggestion](https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1324#issuecomment-462767363) was to remove `*f_cell` in the calculation `CLSpec=f(CL_pls,...)`. Thus, same CLSpec in plasma and in a enzymatic process would still lead to the same clearance rate.

**Comment by msevestre** [comment]
Even better then :)

**Comment by teutonicod** [comment]
@msevestre, what I meant was to correct the formula as suggested by @sfrechen , but I was also wandering if we should leave the naming _Plasma clearance_ as it is, since from my point of view, this is not clear at all. 

From my understanding, the parameter _Plasma clearance_ is a back calculation, where a total hepatic clearance is used to calculate the intrinsic clearance, according to the equation mentioned by @Christoph27 (Yang, J., et al., Drug Metabolism and Disposition 35.3 (2007): 501-502). Checking this publication, I think that this input parameter would be actually a total liver clearance, so not sure that _Plasma clearence_ is the best way to refer to such parameter.

**Comment by msevestre** [comment]
>From my understanding, the parameter Plasma clearance is a back calculation,

I have the feeling we are not talking about the same parameter lol. the Plasma Clearance is an input.  This is not derived in any shape or form from other parameters

> I think that this input parameter would be actually a total liver clearance, so not sure that Plasma clearence is the best way to refer to such parameter.

No opinion on that. This was named by our experts so I don't want to open a can of worms here :)

**Comment by teutonicod** [comment]
@msevestre, I agree, I also had the feeling we were talking about two different parameters (thatś why I clrarly stated the name of the parameter I was referring to ;-)). But this was the parameter my post was originally about, so I was hoping we could also solve this issue :-) (if we think that there is an issue to solve, of course).

Let's see what the other members think about the plasma clearance. I also agree that it does not have any impact, so also wondering on the utility of such parameter.

Also, I would suggest that we mention clearly which equation are we suggesting to change, so that we are all on the same page.

**Comment by msevestre** [comment]
@teutonicod 
![image](https://user-images.githubusercontent.com/1041237/66852897-03b11280-ef4c-11e9-9d02-1eab81dbcf19.png)

This is an INPUT parameter. No backcalculation. If this is the parameterm you mean, then I don't understand what you mean :)

**Comment by teutonicod** [comment]
Sorry, but I do not have PK-Sim with me, so I may have confused the parameters. I will check that tomorrow.

**Comment by sfrechen** [comment]
From my point of view, we just need to ajust the formula with regard to back calculation.
Naming etc. is fine for me. The process' name tells us that we assume that total plasma clearance equals hepatic plasma clearance.

Maybe a tooltip could be helpful to inform the user about the wll-stirred back calculation and its assumptions (but not necessary...).

**Comment by Christoph27** [comment]
I agree with Sebastian regarding naming and a possible tooltip

**Comment by teutonicod** [comment]
> The process' name tells us that we assume that total plasma clearance equals hepatic plasma clearance.

From my point of view this was not very clear. But if it is just me, we can leave it as it is. Nevertheless I would strongly suggest updating the tooltip of the parameter Plasma clearence itself, since for now this is what we have :-)


![image](https://user-images.githubusercontent.com/11489621/66897287-f4d36a00-eff6-11e9-96e7-c5434d34909a.png)

**Comment by dzianismr** [comment]
Hi Folks,

I was looking into the descrepancy between inputed and calculated CL in PK-Sim, when I came accross this thread. It looks like @sfrechen pinpoint the issue to  f_cell in the formula. However PK-Sim 11 still has the same formular. Any thoughts/updates on the topic?

**Comment by PavelBal** [comment]
@sfrechen @Yuri05 @msevestre 

Should we implement the "fix" by removing the `CP` from the equation? I would actually get rid of the "Unspecific liver" enzyme completely and define the reaction with container criteria.

This will of course be a "breaking change" as old models will behave differently (higher clearance).

**Comment by PavelBal** [comment]
@sfrechen I think we are suggesting the same, just in the different steps... The liver clearance reaction rate is given as

1. `Clearance rate = CP * CLspecPerEnzyme * M * K_water`,

with `CP` being the concentration of the `Unspecific liver enzyme`, which is `refConc * f_cell = 1/f_cell` because the reference concentration is always 1. So we get

2. `Clearance rate = 1/f_cell * (CLspec / EnzymeConcentration) * M * K_water`

where `EnzymeConcentration` is the **input parameter** (by default 1 µmol/l)

3.
```
Clearance rate = 1/f_cell * (
CL_pls * BW * (Q_liv+Q_pve) / (fu  * (Q_liv+Q_pve - CL_pls / B2P* BW) * V * f_cell))
 / EnzymeConcentration(=1)) * M * K_water
```

I propose to remove `CP` from 1., you propose to remove `f_cell` from 3. I favor 1. because then we could remove the `Unspecific liver enzyme` completely from the model.

To prove that this correction makes sense:
1. I Opened the Alfentanil project and replaced the GFR and enzyme-mediated clearance with Liver plasma clearance. When I enter `3.5 ml/min/kg` as input clearance value and simulate, the calculated plasma clearance is 4.6:

![image](https://user-images.githubusercontent.com/25903040/226439949-990dcd00-bf04-4d2d-8c33-58c1820a171a.png)


1. I corrected the input by dividing by 1.5, the value of `CP` in the default model 3.5 / 1.49925037481259 = 2.3345. Now the calculated plasma clearance is 3.2 which is much closer to "3.5" what I actually wanted to have.

**Comment by sfrechen** [comment]
@PavelBal 
Ok, sorry, my mistake. I just considered CP as "Clearance plasma" without really checking what it actually is. Thanks for clarifying. Indeed, both suggestions are then structurally identifical.
I leave it to the developer team to decide :)

**Comment by ahmedsanot2009** [comment]
in case of compounds eliminated hepatically and renally, the plasma clearance as input for unspecified total liver clearance will not be the same as total plasma clearance.  Iam I right?

**Comment by PavelBal** [comment]
@Yuri05 I think this should be implemented.
